sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java

```diff
@@ -32,18 +33,25 @@
 import javax.inject.Inject;
 import org.sormas.e2etests.helpers.WebDriverHelpers;
 import org.sormas.e2etests.pojo.web.Event;
+import org.sormas.e2etests.pojo.web.EventGroup;
 import org.sormas.e2etests.pojo.web.Person;
+import org.sormas.e2etests.services.EventGroupService;
 import org.sormas.e2etests.services.EventService;
 
 public class EditEventSteps implements En {
 
   private final WebDriverHelpers webDriverHelpers;
   public static Event event;
+  public static EventGroup groupEvent;
   public static Person person;
   public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("M/d/yyyy");
 
   @Inject
-  public EditEventSteps(WebDriverHelpers webDriverHelpers, EventService eventService, Faker faker) {
+  public EditEventSteps(
+      WebDriverHelpers webDriverHelpers,
+      EventService eventService,
+      Faker faker,
```

Comment:
where are we using this object here?

Reply:
you mean the faker ? at line 126 127
          webDriverHelpers.fillInWebElement(PARTICIPANT_FIRST_NAME_INPUT, faker.name().firstName());
          webDriverHelpers.fillInWebElement(PARTICIPANT_LAST_NAME_INPUT, faker.name().lastName());

Reply:
wouldn't be best to keep the faker in POJOs and collect data from there? :D

Reply:
was added with a different flow, since editing deleting participants was not handled yet pojo and service for participants was no created yet

Reply:
understood. No problem, we will need to add them as well because they will be used in future for other tests. If you don't have time to address and change them, no worries, just let me know and I'll pick up this PR

Reply:
i agree but since there are no advanced tests yet regarding participants at least not according to this planning, maybe when its addressed in the future it can be added
![image](https://user-images.githubusercontent.com/83635257/121545191-94364380-ca0a-11eb-8261-20ca4d792207.png)